src/HyeEun/filter/components/common/OptionButton/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+
+import { St } from "./style";
+
+interface OptionbuttonProps {
+  content: string;
+  submitSet: Set<string>;
+  setSubmitSet: React.Dispatch<React.SetStateAction<Set<string>>>;
+}
+
+export default function OptionButton(props: OptionbuttonProps) {
+  const { content, submitSet, setSubmitSet } = props;
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleClick = (content: string) => {
+    setIsClicked(!isClicked);
+    // submitSet에 content가 있으면 submitSet에서 삭제, 아니면 추가
+    const tempSubmitSet = new Set([...submitSet]);
+    tempSubmitSet.has(content) ? tempSubmitSet.delete(content) : tempSubmitSet.add(content);
+  };
```

Comment:
```suggestion
    const tempSubmitSet = new Set([...submitSet]);
    tempSubmitSet.has(content) ? tempSubmitSet.delete(content) : tempSubmitSet.add(content);
    setSubmitSet(tempSubmitSet);
  };
```

왜 안 되나 했더니 setState 함수가 빠졌네욥하하하